modules/math/src/main/java/com/opengamma/strata/math/impl/statistics/descriptive/QuantileCalculationMethod.java

```diff
@@ -16,18 +17,30 @@
    * Compute the quantile estimation.
    * <p>
    * The quantile level is in decimal, i.e. 99% = 0.99 and 0 < quantile < 1 should be satisfied.
```

Comment:
Should clarify if the 99% is from top or bottom, i.e. Quantile Q such that sample[i]<Q for 99% of the i or sample[i]>Q for 99% of the i?